fusionbackend/src/main/java/io/fusion/fusionbackend/model/BaseAsset.java

```diff
@@ -33,6 +33,8 @@
     protected String imageKey;
 
     public void copyFrom(final BaseAsset sourceBaseAsset) {
+        super.copyFrom(sourceBaseAsset);
```

Comment:
beim Rest hattest du auch 1 Leerzeile davor

Reply:
Ich sehe im Code dutzende Stellen in denen mal eine Leerzeile ist und mal nicht. Ich schlage vor, dass das über eine Code-Formatter-Regel, wenn möglich, einstellen und nicht einzeln in Reviews das korrigieren. Setzt voraus, dass der Code-Formatter im Team geteilt wird.